### DISCUSSION TITLE
how to push changes from visual studio code?

### DISCUSSION THREAD

**Discussion by HackerstoyanovBG** [community/community#35766] [New to GitHub] [answered]
how to push changes from visual studio code?

**Comment by MitkoVtori** [comment, marked as answer]
![Screenshot 2022-10-11 143537](https://user-images.githubusercontent.com/112943652/195086435-d4338fa1-70d2-4210-a9e3-46189186eb72.png)
![Screenshot 2022-10-11 143817](https://user-images.githubusercontent.com/112943652/195086459-3fbc8696-f1a7-4c6c-9487-9c2b29050d54.png)
**Or you can use commands directly from here:**
![Screenshot 2022-10-11 151600](https://user-images.githubusercontent.com/112943652/195087142-8cf7b4a0-3090-417b-a865-6a6f50635259.png)
